.github/workflows/python-package.yml

```diff
@@ -0,0 +1,46 @@
+# Workflow to send master to pypi and tag  the branch:
+# You need to edit FOLDER_WITH_VERSION with the folder that has the __version__ value. 
+
+name: master to pypi with comments and tag
+
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+
+env:
+  FOLDER_WITH_VERSION: dynamo
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine 
+        pip install scipion-pyworkflow 
+        pip install scipion-em
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/* -c "${{ secrets.PYPI_COMMENT }}"
+    - name: Get version and tag
+      env:
+        FOLDER_WITH_VERSION: dynamoo
```

Comment:
Shouldn't it be "dynamo" instead of "dynamoo" in this line?

Reply:
dynamo!!!!! my mistake .... I have already fixed it